Documentation/InputSimulation/InputSimulationService.md

```diff
@@ -69,58 +76,58 @@ The input simulation supports emulated hand devices. These virtual hands can int
 
 ### Hand simulation mode
 
-In the [input simulation tools window](#input-simulation-tools-window) the __Hand Simulation Mode__ setting switches between two distinct input models. The default mode can also be set in the input simulation profile.
+In the [input simulation tools window](#input-simulation-tools-window) the **Hand Simulation Mode** setting switches between two distinct input models. The default mode can also be set in the input simulation profile.
 
-* _Articulated Hands_: Simulates a fully articulated hand device with joint position data.
+* *Articulated Hands*: Simulates a fully articulated hand device with joint position data.
 
    Emulates HoloLens 2 interaction model.
 
    Interactions that are based on the precise positioning of the hand or use touching can be simulated in this mode.
 
-* _Gestures_: Simulates a simplified hand model with air tap and basic gestures.
+* *Gestures*: Simulates a simplified hand model with air tap and basic gestures.
 
    Emulates [HoloLens interaction model](https://docs.microsoft.com/windows/mixed-reality/gestures).
 
-   Focus is controlled using the Gaze pointer. The _Air Tap_ gesture is used to interact with buttons.
+   Focus is controlled using the Gaze pointer. The *Air Tap* gesture is used to interact with buttons.
 
 ### Controlling hand movement
 
-Press and hold the __Left/Right Hand Control Key__ (default: *Left Shift* for left hand and *Space* for right hand) to gain control of either hand. While the manipulation key is pressed, the hand will appear in the viewport. Once the manipulation key is released, the hands will disappear after a short __Hand Hide Timeout__.
+Press and hold the **Left/Right Hand Control Key** (default: *Left Shift* for left hand and *Space* for right hand) to gain control of either hand. While the manipulation key is pressed, the hand will appear in the viewport. Once the manipulation key is released, the hands will disappear after a short **Hand Hide Timeout**.
 
-Hands can be toggled on permanently in the [input simulation tools window](#input-simulation-tools-window) or by pressing the __Toggle Left/Right Hand Key__ (default: *T* for left and *Y* for right). Press the toggle key again to hide the hands again.
+Hands can be toggled on permanently in the [input simulation tools window](#input-simulation-tools-window) or by pressing the **Toggle Left/Right Hand Key** (default: *T* for left and *Y* for right). Press the toggle key again to hide the hands again.
 
-Mouse movement will move the hand in the view plane. Hands can be moved further or closer to the camera using the __mouse wheel__.
+Mouse movement will move the hand in the view plane. Hands can be moved further or closer to the camera using the **mouse wheel**.
```

Comment:
Aside: I wish this was customizable, like I could press ctrl + up to move forward, or 'i' to move forward the hand. It's hard to scroll on my mac :-)

Reply:
100. Especially it was annoying for keyboards that aren't US based. You can re-route input keys in Unity I believe and the input simulation profile but I can't recall how extensive it is